lib/translations_checker/checker.rb

```diff
@@ -70,12 +72,20 @@ def translated_locales
       @translated_locales ||= locales - %w(en en-AU)
     end
 
+    def git_diffs
+      GitDiff.call(LOCALES_DIR)
+    end
+
     def diffs
-      @diffs ||= GitDiff.call(LOCALES_DIR).map do |path, hunks|
+      @diffs ||= supported_git_diffs.map do |path, hunks|
         Diff.new(path, hunks)
       end
     end
 
+    def supported_git_diffs
+      git_diffs.select{|path, hunks| SUPPORTED_FORMATS.include?(path.split('.')[-1])}
```

Comment:
Put spaces around the `{` and `}` and use `last` instead of `[-1]`:
```ruby
git_diffs.select { |path, hunks| SUPPORTED_FORMATS.include?(path.split('.').last) }
```